packages/babel-node/src/babel-node.js

```diff
@@ -92,7 +92,7 @@ getV8Flags(function(err, v8Flags) {
         if (signal) {
           process.kill(process.pid, signal);
         } else {
-          process.exit(code);
+          process.exitCode = code;
```

Comment:
node.js will terminate when all `exit` listener is executed, so we don't need to call `process.exit` again in the listener.